test/cmdlineTests/standard_model_checker_divModSlacks_default_all/output.json

```diff
@@ -0,0 +1,49 @@
+{"auxiliaryInputRequested":{"smtlib2queries":{"0x10763dfdad96614fe8fcaf54161ed2be500bbaa57bda8e2d706b675f1290f13b":"(set-option :produce-models true)
```

Comment:
A few remarks not completely related to this PR:
- `auxiliaryInputRequested`
    - The field is not documented as a part of Standard JSON output in `Using the compiler`.
    - CLI does not produce this output. Why?
    - I see that CHC does not produce this output either. Why?
- `solvers` option added in #11421 is not listed in Standard JSON input either. It's only on the page about SMT.

Reply:
BTW, if any of this needs fixing, it does not necessarily have to be done in this PR (unless it's this PR that broke it).

Reply:
Right, this `auxiliaryInputRequested` is a bit of legacy that I'm not sure was ever used. Back when we didn't have emscripten z3 embedded into the compiler, we used this double run where you run the compiler once, get the queries like this, run a local solver, and input the results back into another compiler run via `auxiliaryInput`, using the same query hashes.
CLI doesn't produce it because the feature is only available via JSON.
CHC does produce it as well, when it can't prove a property.

Reply:
Ah ok, `solver option` should be there, so will open another PR fixing that later.

Reply:
OK, that explains it. What are your plans for `auxiliaryInputRequested`? Are you going to eventually remove it or can it be still useful in some rare cases?

Reply:
We talk about removing it in a call, but didn't reach consensus. Maybe we should just remove it